### ISSUE TITLE
[BUG] Missing peers on Peers page with multiple tunnels with duplicate peers

### ISSUE THREAD

**Issue by brandonm** [theonemcdonald/pfSense-pkg-WireGuard#118]
**Describe the bug**
I have two tunnels on Mullvad setup as interfaces.  They have the same peers added to both. When I view the Peers page it shows them all but when I click on Status tunnel 1 shows all peers but tunnel 2 only shows one peer.

**To Reproduce**
Add two tunnels with the same peers

**Expected behavior**
I expect to see all peers for both tunnels on the Peers page.

**Screenshots**
[Screenshot of Peers Page](https://imgur.com/Nplz2fi)
[Screenshot of Status Page](https://imgur.com/zjzr5qy)

**Version information (please complete the following information):**
 - PfSense: 21.05-RELEASE
 - WireGuard-tools: 1.0.20210424
 - WireGuard-kmod: 0.0.20210503
 - pfSense-pkg-WireGuard: 0.1.1

**Comment by theonemcdonald** [comment]
This will need to be reevaluated after the latest code is available on pfSense+ The version you are running is old and a lot has changed on the backend

**Comment by theonemcdonald** [comment]
@brandonm can you tell me exactly how you have this setup? I have Mullvad, so I'd like to replicate your setup with my own keys

**Comment by brandonm** [comment]
It's pretty standard setup.  I removed the old pfSense Wireguard Interfaces and tunnels and upgraded to 21.05-RELEASE. After rebooting I installed your new Wireguard package from the Package Manager.

I created one tunnel and added all Mullvad San Jose, CA servers servers 195 to 208 as peers.  I only added IPv4 with Allowed IP 0.0.0.0/0.

The name for the tunnels are "Mullvad (San Jose, CA) - 1" and "Mullvad (San Jose, CA) - 2" and the peer names are "mullvad-us195 (san jose, ca)" the numer is the actual server number for each.  It's the same peer names and settings on both tunnels (After adding I tried to change so they were unique but that didn't fix it)

I added Interfaces for each named WG_MULLVAD and WG_MULLVAD2.  Then modified my NAT and Firewall rules that were existing from the old Wireguard tunnels just needed to update the Gateways.

Oh one thing I did try when having to manually add all these peers, was to log into the ssh and as a test edited /usr/local/etc/wireguard/tun_wg0.conf and changed the name of one of the tunnels for example "mullvad-us195 (san jose, ca)" to "mullvad-us195 (san jose, ca) a" to see if it would reflect but after restarting the Wireguard and even stopping before editing it was just overwritten.  I ended up just manually entering them, not sure if that did something.
